aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -955,6 +961,126 @@ void qtopk_kernel(Tensor& values,
   });
 }
 
+template <bool ReluFused>
+void q_batch_norm_kernel(
+    int N,
+    int C,
+    int HxW,
```

Comment:
We sure we want to use `int` for all these shape parameters? It's probably better if this is at least fixed-width, and preferrably `int64_t` to match the rest of the index handling in pt